core/src/main/java/com/graphhopper/routing/util/spatialrules/SpatialRuleLookupHelper.java

```diff
@@ -41,7 +41,7 @@
 public class SpatialRuleLookupHelper {
 
     private static final Logger logger = LoggerFactory.getLogger(SpatialRuleLookupHelper.class);
-    static String JSON_ID_FIELD = "ISO_A3";
+    public static String JSON_ID_FIELD = "ISO3166-1:alpha3";
```

Comment:
This change should be included in the changelog. 

And is this used elsewhere or why did you make it public? E.g. it could be used in Measurement?

Reply:
Probably a relict from my other PR.

Reply:
Thanks! IMO it should be still included in the changelog. GeoJSON files with ISO_A4 will now break (?)

Reply:
Done